infra/recipes/terraform/azure/modules/helm_release/main.tf

```diff
@@ -0,0 +1,69 @@
+locals {
+  k8s_path = "../../../../k8s"
```

Comment:
Can we determine the path prefix in a consistent way using ${abspath(path.module)}?

Reply:
Yes

Reply:
Hi, I actually tested this and the terraform `file` function that reads the path needs a relative path here, which doesn't work on the absolute path. Thus, I am keeping it as the relative path for now.